plugins/modules/ipahbacrule.py

```diff
@@ -188,13 +188,9 @@ def find_hbacrule(module, name):
     elif len(_result["result"]) == 1:
         res = _result["result"][0]
         # hbacsvcgroup names are converted to lower case while creation with
-        # hbacsvcgroup_add.
```

Comment:
Please readd the comment or provide a better one :-) Sudo is old and an exception.

Reply:
Fixed.